matrix/instrumentations.yaml

```diff
@@ -0,0 +1,303 @@
+- keys:
+    - ASPNET
+  library: ASP.NET Framework (.NET Framework)
+  version: See :ref:`dotnet-otel-versions`
+  comment: MVC / WebApi (Only integrated pipeline mode supported). Metrics requires trace instrumentation.
```

Comment:
```suggestion
  description: MVC / WebApi (Only integrated pipeline mode supported). Metrics requires trace instrumentation.
```

Reply:
Fabrizio? what is your preference to the naming?

Reply:
You are already using `description` e.g. in metrics description. I find `description` more general and usable. Comment is rather an addition to an existing description.

Reply:
+1, let's use `description`